src/Sylius/Bundle/ShopBundle/Resources/views/Order/thankYou.html.twig

```diff
@@ -22,10 +22,13 @@
                     </div>
                 {% endif %}
                 <div class="ui hidden divider"></div>
-                <a href="{{ path('sylius_shop_order_show', {'tokenValue': order.tokenValue}) }}" id="sylius-show-order" class="ui large blue button">{{ 'sylius.ui.view_your_order_or_change_payment_method'|trans }}</a>
+                {% if order.customer.user is not null %}
```

Comment:
Could we move that `if` outside the `if` with `lastPayment != false`? We can group it together with the other one below and use `if` and `else`.